inst/eDISH_app/modules/renderSettings/renderSettings.R

```diff
@@ -140,34 +142,16 @@ renderSettings <- function(input, output, session, data, settings, status){
   ######################################################################
   
   settings_new <- reactive({
+   
+    keys<-input_names() %>% as.character()
+    values<- keys %>% map(~ifelse(is.null(input[[.x]]) , "", input[[.x]])) #Only capturing the first value in vectors ... 
+    inputDF <- tibble(text_key=keys, customValue=values) %>% mutate(customValue= unlist(customValue))
```

Comment:
nice! Is the goal that we allow "values" to be a list-column so that all the values can be collected? E.g.

```
    getInputs <- function(x){
      if (is.null(input[[x]])){
        return("")
      } else{
        return(input[[x]])
      }
    }
    values<- keys %>% map(~getInputs(.x))
    inputDF <- tibble(text_key=keys, customValue=values) 
```